src/pudl/extract/eia176.py

```diff
@@ -1,26 +1,34 @@
-"""Extract EIA Form 176 data from CSVs.
+"""Extract EIA Form 176 data from CSVs."""
 
-The EIA Form 176 archive also contains CSVs for EIA Form 191 and EIA Form 757.
-"""
+from dagster import AssetOut, Output, multi_asset
 
-from dagster import asset
+from pudl.extract.csv import CsvExtractor
+from pudl.extract.extractor import raw_df_factory
 
-from pudl.extract.csv import CsvExtractor, get_table_file_map
+raw_table_names = "raw_eia176__data"
 
-DATASET = "eia176"
 
+raw_eia176__all_dfs = raw_df_factory(CsvExtractor, name="eia176")
 
-@asset(required_resource_keys={"datastore"})
-def raw_eia176__company(context):
-    """Extract raw EIA company data from CSV sheets into dataframes.
 
-    Args:
-        context: dagster keyword that provides access to resources and config.
+@multi_asset(
```

Comment:
Multi-assets get cranky when there's only one asset (they wind up making an asset for each letter of the string in `raw_table_names`. This might change a bit depending on how the extractor gets refactored, but it currently should look something closer to:

```suggestion
from dagster import AssetOut, Output, asset
from pudl.extract.csv import CsvExtractor
from pudl.extract.extractor import raw_df_factory


class Eia176Extractor(CsvExtractor):
    """Extractor for EIA form 176."""

    def __init__(self, *args, **kwargs):
        """Initialize the module.

        Args:
            ds (:class:datastore.Datastore): Initialized datastore.
        """
        self._dataset_name = "eia176"
        super().__init__(dataset_name= self._dataset_name, *args, **kwargs)

raw_eia176__all_dfs = raw_df_factory(Eia176Extractor, name="eia176")

@asset(
    required_resource_keys={"datastore", "dataset_settings"},
)
def raw_eia176__data(context, raw_eia176__all_dfs):
    """Extract raw EIA company data from CSV sheets into dataframes.

    Returns:
        An extracted EIA 176 dataframe.
    """
    return (
        Output(value=df)
        for df in raw_eia176__all_dfs.items()
    )
```